Documented information about sleep and wake namespace (#366)
okteto/docs: src/content/reference/cli.mdx

 ✓  Using context cindy @ cloud.okteto.com
```

#### sleep

> Sleeps the okteto namespace. 

Reviewer: Could we make it optional? that way, `okteto namespace sleep` sleeps the current namespace in my okteto context

```console
$ okteto namespace sleep test-cindy
```

```console
i  Using test-cindy @ cloud.okteto.com as context

> ✓  Namespace 'test-cindy' is sleeping

Reviewer: does the cli wai for the sleep operation to finish?

Author: It waits for 1-2 sec.

Author: Do we need to make any changes?

Reviewer: No, not here

```

#### wake

Wakes the sleeping okteto namespace

```console
$ okteto namespace wake test-cindy
```

```console
i  Using test-cindy @ cloud.okteto.com as context

> ✓  Namespace 'test-cindy' is awake now

Reviewer: does the cli wai for the wake operation to finish?

Author: It waits for 1-2 sec.

 ✓  Using context cindy @ cloud.okteto.com
```

#### sleep

> Sleeps the default okteto namespace. 

Reviewer: ```suggestion
Sleeps an okteto namespace. By default, it sleeps the current okteto namespace. 
```

```console

> $ okteto namespace sleep

Reviewer: ```suggestion
$ okteto namespace sleep [name]
```

```

```console
i  Using test-cindy @ cloud.okteto.com as context
✓  Namespace 'test-cindy' is sleeping
```

> If you'd like to sleep a namespace other than default, you can provide that namespace name as an argument to the command.

Reviewer: ```suggestion
If you'd like to sleep a namespace other than the current one, you can provide that namespace name as an argument to the command.
```

This will look something like `okteto namespace sleep <namepace-name>`.

#### wake

> Wakes the default okteto namespace

Reviewer: ```suggestion
Wakes an okteto namespace. By default, it wakes the current okteto namespace.
```

```console

> $ okteto namespace wake

Reviewer: ```suggestion
$ okteto namespace wake [name]
```

```

```console
i  Using test-cindy @ cloud.okteto.com as context
✓  Namespace 'test-cindy' is awake now
```

> If you'd like to wake a namespace other than default, you can provide that namespace name as an argument to the command.

Reviewer: ```suggestion
If you'd like to wake a namespace other than the current one, you can provide that namespace name as an argument to the command.
```
